doc/nrf/gs_modifying.rst

```diff
@@ -198,25 +319,33 @@ Creating build type files
 
 To create custom build type files for your application instead of using a single :file:`prj.conf` file, complete the following steps:
 
-1. During :ref:`application development <zephyr:application>`, follow the procedure for creating the application until after the step where you create the :file:`CMakeLists.txt` file.
-#. In the :file:`CMakeLists.txt` file, define the file name pattern for configuration files.
-   For example::
+1. Create a new :file:`prj_buildtype.conf` in the main application folder next to the :file:`prj.conf` file, with *buildtype* in the file name corresponding to your build type name.
```

Comment:
Will `prj_<buildtype>.conf` render correctly ?

If it will, then I think `buildtype` should be surrounded with `<` and `>` to indicate it's just a placeholder for whatever name the user chooses.